src/services/fetchers/ByBitFetcher.ts

```diff
@@ -0,0 +1,103 @@
+import axios from 'axios';
+import {inject, injectable} from 'inversify';
+import {Logger} from 'winston';
+
+import Settings from '../../types/Settings.js';
+
+type FeedValue = {fsym: string; tsym: string};
+
+export type InputParams = Map<string, FeedValue>;
```

Comment:
if we exporting it, lets do 
```suggestion
export type ByBitInputParams = Map<string, FeedValue>;
```

Reply:
Done